test/unit/serialize.js

```diff
@@ -424,6 +424,29 @@ describe('serialize( obj )', function () {
             expect(serialize(["<"], {space: 2})).to.equal('[\n  "\\u003C"\n]');
             expect(serialize(["<"], {unsafe: true, space: 2})).to.equal('[\n  "<"\n]');
         });
+
+        it("should accept a `ignoreFunction` option", function() {
+            function fn() { return true; }
+            const obj = {
+              fn,
+              fn_arrow: () => {
+                return true;
+              }
+            };            
+            const obj2 = {
+                num:123,
+                str:'str',
+                fn
+            }
+            // case 1. Pass function to serialize
+            expect(serialize(fn, { ignoreFunction: true })).to.equal(`undefined`);
```

Comment:
nit: back-quotes are unnecessary?
```suggestion
            expect(serialize(fn, { ignoreFunction: true })).to.equal('undefined');
```